zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/repo/VFSNotebookRepoTest.java

```diff
@@ -45,22 +45,21 @@ class VFSNotebookRepoTest {
   private ZeppelinConfiguration zConf;
   private NoteParser noteParser;
   private VFSNotebookRepo notebookRepo;
-  private File notebookDir;
```

Comment:
I removed this field. I thought the `notebookRepo.rootNotebookFolder` would be enough to check notebook directory.
It is also related to the issue that a directory is created every time the test is run.